ooniprobe/AppDelegate.mm

```diff
@@ -25,7 +23,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     [Fabric with:@[[Crashlytics class]]];
     #endif
 
-    [[UIApplication sharedApplication] setStatusBarStyle:UIStatusBarStyleLightContent];
+    application.statusBarStyle = UIStatusBarStyleLightContent;
```

Comment:
Here you're using the property approach rather than the message dispatching one, correct?